### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

### CODE REVIEW

File: pandas/core/indexes/base.py [hunk 1]
```diff
@@ -1221,6 +1221,18 @@ def copy(
         -----
         In most cases, there should be no functional difference from using
         ``deep``, but if ``deep`` is passed it will attempt to deepcopy.
+
+        Examples
+        --------
+        >>> idx = pd.Index(['a', 'b', 'c'])
+        >>> new_idx = idx.copy(name="idx2")
+        >>> ser = pd.Series([1, 2, 3], index=new_idx)
```
Comment (mroeschke) [inline pandas/core/indexes/base.py:1229]: Probably better to show that `idx is new_idx` (without changing the `name`) returns `False` instead

**Review by MarcoGorelli — APPROVED** [review]
nice one, thanks @DeaMariaLeon !

EDIT: oops, docs build ended up not being green

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
changes look good, but looks like there's some CI failures

```
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.itemsize'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.na_value'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.names'. Check your autosummary_generate setting.
looking for now-outdated files... none found
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.numpy_dtype'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.construct_array_type'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.construct_from_string'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.empty'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.from_numpy_dtype'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.is_dtype'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float32Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float32Dtype.type'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.itemsize'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.na_value'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.names'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:33: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.numpy_dtype'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.construct_array_type'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.construct_from_string'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.empty'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.from_numpy_dtype'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.is_dtype'. Check your autosummary_generate setting.
/home/runner/work/pandas/pandas/pandas/core/arrays/floating.py:docstring of pandas.core.arrays.floating.Float64Dtype:51: WARNING: autosummary: stub file not found 'pandas.Float64Dtype.type'. Check your autosummary_generate setting.
```

File: pandas/core/arrays/floating.py [hunk 2]
```diff
@@ -127,13 +127,19 @@ class FloatingArray(NumericArray):
 
 This dtype uses ``pd.NA`` as missing value indicator.
 
-Attributes
-----------
-None
```
Comment (MarcoGorelli) [inline pandas/core/arrays/floating.py:132]: this one (and methods) might need to stay

Comment (DeaMariaLeon) [reply pandas/core/arrays/floating.py]: Done, thanks

**Review by MarcoGorelli — APPROVED** [review]
cool, let's get this in, thanks @DeaMariaLeon !

**Pull request merged**